openslides/assignments/static/js/assignments/site.js

```diff
@@ -40,6 +40,9 @@ angular.module('OpenSlidesApp.assignments.site', [
                     assignments: function(Assignment) {
                         return Assignment.findAll();
                     },
+                    users: function(User) {     //TODO: Load only related users cause of better performance
+                        return User.findAll();
+                    },
```

Comment:
`openslides-libs.js:5 Error: assignments/relateduser.inject: "attrs" must be an object or an array!
Error`